app/models/statuses.py

```diff
@@ -5,9 +5,9 @@ def get_formatted(value: str):
     statuses = {    
     "NOT_STARTED": gettext("Not Started"),
     "IN_PROGRESS": gettext("In Progress"),
-    "COMPLETED": gettext("Assessment complete"),
-    "QA_COMPLETED": gettext("QA complete"),
-    "FLAGGED": gettext("Flagged"),
-    "STOPPED": gettext("Stopped"),
+    "COMPLETED": gettext("Completed"),
```

Comment:
Did we mean to change e.g. Assessment complete to Completed? CC @Adam-W1

Reply:
Hi @gidsg. Adam has already pushed the requested changes. Are you ok to re review it please?

Adam is on holidays so I @ramsharma-prog  looking after this PR.